diff -urN supertux-0.3.0/supertux.desktop supertux-0.3.0.new/supertux.desktop
--- supertux-0.3.0/supertux.desktop	2006-12-18 00:22:23.000000000 +0100
+++ supertux-0.3.0.new/supertux.desktop	2006-12-30 17:01:56.000000000 +0100
@@ -13,6 +13,7 @@
 GenericName[en]=A Platform Game
 GenericName[de]=Ein Jump&Run Spiel
 GenericName[cs]=Plošinová hra
+GenericName[pl]=Gra platformowa
 GenericName[nn]=Plattformspel
 GenericName[sv]=Ett plattformsspel
 GenericName[pt_BR]=Jogo de Plataforma
@@ -20,6 +21,7 @@
 Comment[en]=A classic 2D platform game
 Comment[de]=Ein von Super Mario inspiriertes Pinguin Jump&Run Spiel
 Comment[cs]=Plošinovka inspirovaná SuperMariem s tučňákem Tuxem v hlavní roli
+Comment[pl]=Gra podobna do oryginalnej gry Super Mario Bros
 Comment[nn]=Eit klassisk todimensjonalt plattformspel.
 Comment[sv]=Ett klassiskt tvådimensionellt plattformsspel
 Comment[pt_BR]=Encarne o pinguin Tux neste jogo inspirado em clássicos de Pular&Correr
@@ -28,3 +30,4 @@
 Terminal=false
 StartupNotify=false
 Categories=Game;ArcadeGame;
+# vi: encoding=utf-8
