diff -urN aa/supertux-0.0.6/configure.ac supertux-0.0.6/configure.ac
--- aa/supertux-0.0.6/configure.ac	2004-03-15 19:21:01.000000000 +0100
+++ supertux-0.0.6/configure.ac	2004-03-17 17:22:04.268024160 +0100
@@ -103,7 +103,7 @@
 fi
 
 # FIXME: Evil
-CFLAGS="$CFLAGS -DDATA_PREFIX='\"$datadir/supertux/\"' -DLINUX"
+CFLAGS="$CFLAGS -DDATA_PREFIX='\"$datadir/games/supertux/\"' -DLINUX"
 
 dnl Checks for library functions.
 AC_CHECK_FUNCS(mkdir strdup strstr)
