--- supertux-0.3.1/supertux2.desktop~	2008-01-03 21:58:48.000000000 +0100
+++ supertux-0.3.1/supertux2.desktop	2008-01-07 00:16:29.000000000 +0100
@@ -16,6 +16,7 @@
 GenericName[it]=Videogioco a Piattaforme
 GenericName[lt]=Platformos žaidimas
 GenericName[nb]=Plattformspill
+GenericName[pl]=Gra platformowa
 GenericName[nn]=Plattformspel
 GenericName[nl]=Platformspel
 GenericName[pt]=Jogo de Plataforma
@@ -31,6 +32,7 @@
 Comment[fi]=Ohjaa pingviiniä perinteisessä tasohyppelyssä
 Comment[lt]=Klasikinis 2D platformos žaidimas
 Comment[nb]=Et klassisk todimensjonalt plattformspill.
+Comment[pl]=Gra podobna do oryginalnej gry Super Mario Bros
 Comment[nn]=Eit klassisk todimensjonalt plattformspel.
 Comment[sv]=Ett klassiskt tvådimensionellt plattformsspel
 Comment[pt_BR]=Encarne o pinguin Tux neste jogo inspirado em clássicos de Pular&Correr
@@ -40,3 +42,4 @@
 Terminal=false
 StartupNotify=false
 Categories=Game;ArcadeGame;
+# vi: encoding=utf-8
